### DISCUSSION TITLE
Marketplace assets to source build on linux

### DISCUSSION THREAD

**Discussion by evoevo07** [carla-simulator/carla#7906] [General]
Is it possible to use marketplace assets on a source build carla projekt?
If I want to add some, I am prompt to install a binary version of UE.

**Comment by Blyron** [comment]
It should be possible. What you can do is install the binary version of UE and then migrate from that binary project your CARLA project

**Reply by evoevo07** [reply]
Thanks for the fast reply. Is it possible to migrate from a windows binary to linux source version?

**Reply by Blyron** [reply]
I guess so because they are uassets? But I have never tried that.

**Reply by evoevo07** [reply]
I will try it. Is it a problem if the UE version of the binary is different (5.4)?

**Reply by Blyron** [reply]
If those usassets don't use code from 5.4 version, shouldn't be

**Reply by Blyron** [reply]
But I would try to stick to 5.3 compatible assets

**Reply by evoevo07** [reply]
![Screenshot from 2024-07-04 10-22-08](https://github.com/carla-simulator/carla/assets/164197989/cd86c7cc-8578-48f3-ab36-4712186167c9)

Thanks! it worked :)
